akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardCoordinator.scala

```diff
@@ -690,6 +696,9 @@ abstract class ShardCoordinator(
         })
         sender() ! reply
 
+      case ShardCoordinator.Internal.Terminate =>
+        log.debug("{} received termination message", context.self.path)
```

Comment:
self.path is already included in logging (`akkaSource` MDC if using SLF4J)

other logging here is not including it so let's make it consistent by not including it in the log message

Reply:
Yes, self.path is included, I removed it where needed